content/references/rippled-api/api-conventions/rate-limiting.md

```diff
@@ -0,0 +1,62 @@
+# Rate Limiting
+
+The `rippled` server limits the rate at which API clients can make requests on public APIs. Rate limiting is based on the IP address of the client, so clients behind [network address translation](https://en.wikipedia.org/wiki/Network_address_translation) share a limit based on their public IP address.
+
+**Tip:** Rate limiting does not apply when the client is connected [as an admin](get-started-with-the-rippled-api.html#admin-access).
+
+When a client is approaching the rate limit, the server adds the field `"warning": "load"` at the top level of an [API response](response-formatting.html). This warning is not added to every response, but the server may send several such warnings before it disconnects a client.
+
+If a client goes past the rate limit, the server disconnects that client and does not serve further requests from the client's API address for a while. The disconnect message is different for the WebSocket and JSON-RPC APIs.
```

Comment:
"is different for" - "differs between" (to me, it seems a little clearer)

Reply:
The verb "differs" feels just a little fancy to me, so I changed the sentence around:

> The WebSocket and JSON-RPC APIs use different disconnect messages.